configs/__init__.py

```diff
@@ -34,24 +34,12 @@
     SKALE_VOLUME_PATH = '/skale_vol'
     NODE_DATA_PATH = '/skale_node_data'
 
-
-CONFIG_FOLDER_NAME = 'config'
-CONTRACTS_INFO_FOLDER_NAME = 'contracts_info'
-
-MANAGER_CONTRACTS_INFO_NAME = 'manager.json'
-IMA_CONTRACTS_INFO_NAME = 'ima.json'
-DKG_CONTRACTS_INFO_NAME = 'dkg.json'
-
-CONTRACTS_INFO_FOLDER = os.path.join(SKALE_VOLUME_PATH, CONTRACTS_INFO_FOLDER_NAME)
-CONFIG_FOLDER = os.path.join(SKALE_VOLUME_PATH, CONFIG_FOLDER_NAME)
-
-TOKENS_FILENAME = 'tokens.json'
-TOKENS_FILEPATH = os.path.join(NODE_DATA_PATH, TOKENS_FILENAME)
-
 NODE_CONFIG_FILENAME = 'node_config.json'
 NODE_CONFIG_FILEPATH = os.path.join(NODE_DATA_PATH, NODE_CONFIG_FILENAME)
 
 DEFAULT_SLEEP_TIMEOUT = 5
 
 SGX_KEY_NAME_RETRIES = 30
 SGX_KEY_NAME_TIMEOUT = 20
+
+BLOCKS_TO_WAIT = int(os.getenv('BLOCKS_TO_WAIT', 5))
```

Comment:
I think we need to handle BLOCKS_TO_WAIT = empty string here. Because with high probability someone can set BLOCKS_TO_WAIT='' in .env file

Reply:
addressed 👌